### DISCUSSION TITLE
Metadata dropped when appending HEICs

### DISCUSSION THREAD

**Discussion by alex-seville** [ImageMagick/ImageMagick#7117] [Help]
### ImageMagick version

7.1.1-28

### Operating system, version and so on

MacOSX 14.3.1

### Description

I have 2 heic files, both with some XMP data. When I adjoin them or convert them into a single .heic file only the first Image has the XMP data, the second does not. 

Is there a better way to be combining two HEICs into one to retain that data?

**Comment by dlemstra** [comment]
You will need to do this yourself. We can't and won't add something that combines an image profile. You should script something where you get the two xmp profiles of the image and then combine those. And when you create the single heic files you should remove the xmp profile from the image(s) and add the combine profile yourself.
